File: Zodiac.java
package stembranch;

import java.util.Scanner;

public class Zodiac {
	public static String zodiac(int year) {
		String[] zodiac = { "Êó", "Å£", "»¢", "ÍÃ", "Áú", "Éß", "Âí", "Ñò", "ºï", "¼¦", "¹·", "Öí" };
		return zodiac[(year - 4) % 12];
	}

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int year = scanner.nextInt();
		scanner.close();
		System.out.println(zodiac(year));
	}
}

File: zodiac.py
zodiac = "鼠牛虎兔龙蛇马羊猴鸡狗猪"


def stembranch(year):
    zodiac = "鼠牛虎兔龙蛇马羊猴鸡狗猪"
    return zodiac[(year - 4) % 12]


class StemBranch:
    def __init__(self):
        self.zodiac = "鼠牛虎兔龙蛇马羊猴鸡狗猪"

    def stembranch(self, year):
        zodiac = "鼠牛虎兔龙蛇马羊猴鸡狗猪"
        return zodiac[(year - 4) % 12]
